shell/browser/web_contents_preferences.cc

```diff
@@ -136,7 +136,7 @@ WebContentsPreferences::WebContentsPreferences(
                 "more information",
                 "electron");
   }
-  SetDefaultBoolIfUndefined(options::kContextIsolation, false);
+  SetDefaultBoolIfUndefined(options::kContextIsolation, true);
```

Comment:
you also need to use `IsEnabled(options::kContextIsolation, true);` check #18369 for details